mmv1/products/compute/ansible.yaml

```diff
@@ -284,6 +284,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   InterconnectAttachment: !ruby/object:Overrides::Ansible::ResourceOverride
     tests: !ruby/object:Provider::Ansible::Tests
       has_autogenerated_test: false
+  NodeGroup: !ruby/object:Overrides::Ansible::ResourceOverride
+    properties:
+      initialSize: !ruby/object:Overrides::Inspec::PropertyOverride
```

Comment:
This one should be ansible too, right?

Reply:
yes you are right, sorry about the bad copy job. Kind was on a caffeine craze yesterday and wasn't as careful as I could of been